ember/app/ui/cases/index/template.hbs

```diff
@@ -41,13 +41,15 @@
         />
 
         {{#if (can "filter access case")}}
-          <Filter.Dropdown
-            @label={{t "documents.filters.identity"}}
-            @selected={{this.selectedOptions}}
-            @options={{this.identities}}
-            @onChange={{fn this.updateFilter "selectedIdentities"}}
-            @onInput={{this.updateIdentitySearch}}
-          />
+          <div class="uk-hidden">
```

Comment:
Do you really need the element in the DOM for something? If not, I'd suggest commenting it out instead of only hiding it via CSS

Reply:
Done, thanks!